hibernate-core/src/test/java/org/hibernate/orm/test/query/NativeQueryLimitOffsetTest.java

```diff
@@ -82,6 +83,7 @@ public void testFullLimitOffsetOnNativeQuery(SessionFactoryScope scope) {
 	@SkipForDialect( dialectClass = OracleDialect.class, majorVersion = 12, minorVersion = 2, matchSubTypes = true)
 	@SkipForDialect( dialectClass = AbstractHANADialect.class, matchSubTypes = true)
 	@SkipForDialect( dialectClass = SpannerDialect.class, matchSubTypes = true)
+	@SkipForDialect( dialectClass = AltibaseDialect.class, reason = "Altibase returns 1 2 4 5 7 on first result")
```

Comment:
Could you please add some more details to the reason for the skip? I have no idea what this means.

Reply:
Person table is as follows.

1,John
2,Jack
3,Bob
4,Jill
5,Jane
6,Anne
7,Joe

I don't know exactly the meaning of  testPartialLimitOffsetOnNativeQuery test but actual query is as follows and Oracle is same here.

select id from Person where name like 'J%';
1 2 4 5 7

Reply:
I'm gonna change comments with more reasonable texts.

Reply:
See the comment I left in `AltibaseLimitHandler`. I think you should be able to remove the skip after https://github.com/hibernate/hibernate-orm/pull/7520 is merged and you extend the `LimitLimitHandler`.

Reply:
Yeah. after patching #7520 and modifing AltibaseLimitHandler, I was able to pass this test.